solutionbox/code_free_ml/mltoolbox/code_free_ml/data/analyze_data.py

```diff
@@ -29,6 +29,8 @@
 import textwrap
 
 from tensorflow.contrib import lookup
+from tensorflow.contrib.slim.python.slim.nets.inception_v3 import inception_v3
```

Comment:
No idea. If there are any breaking changes from TF we'll have to change our code.